new-pipeline/src/dashboards.js

```diff
@@ -1081,6 +1081,23 @@ function getDescriptionLink(metric, channel, description) {
       class: "btn btn-outline-primary fa fa-info-circle",
     }).text(" More details"));
   }
-
   return link;
 }
+
+function getUseCounterLink(metric, channel, description) {
+  if (metric.startsWith("USE_COUNTER") !== true) {
```

Comment:
A code style nit: `startsWith()` always returns `true` or `false`. We should just use logical tests here, we don't need to check for a value, i.e. `if (!foo) ...`.

We also only support the current use counters, which should start with `USE_COUNTER2_`.